### PR TITLE
login styles done

### PR DESCRIPTION
**Pull request by jersonlopez**
## Summary
- login styles added
- go to login in the index view when clicking in logout button

## Screenshot
image
login

## Trello Card
https://trello.com/c/Ee0EnDUk/32-login

### CODE REVIEW

File: maquetado/login.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  </head>
+  <body>
+    <div class="login">
+      <img class="logo" src="./assets/LogoWolox.png" alt="wolox-logo">
+      <label for="email" class="text-from">Email</label>
+      <input type="text" id="email" class="input-form">
+      <label for="password" class="text-from">Password</label>
+      <input type="password" id="password" class="input-form">
+      <a href="index.html">
+        <button class="login-button text-from">Login</button>
```
Comment (CrisVega08) [inline maquetado/login.html]: Here you don't need a button you use a tag that is for links so replace it by span.

Remember when i need to execute some logic -> button 
Only redirect => a

File: maquetado/login.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      </a>
+      <hr class="splitter">
```
Comment (CrisVega08) [inline maquetado/login.html:19]: I think you can remove this item and use the border-bottom of the tag above

File: maquetado/login.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <button class="sign-up-button text-from">Sign Up</button>
```
Comment (CrisVega08) [inline maquetado/login.html]: Same about buttons

File: maquetado/scss/application.scss [hunk 4]
```diff
@@ -2,6 +2,7 @@
 @import 'equalizer';
 @import 'index';
 @import 'detail';
+@import'login';
```
Comment (CrisVega08) [inline maquetado/scss/application.scss]:
```suggestion
@import 'login';
```

File: maquetado/scss/login.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: space-between;
+  margin: 180px auto;
+  padding: 25px;
+  width: 300px;
+}
+
+.text-from {
```
Comment (CrisVega08) [inline maquetado/scss/login.scss]: I think this styles already exits, so don't repeat, create a file for text something like text.scss and import it when do you need

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
Nice work only some comments

File: maquetado/login.html (hunk 2 above)
Comment (jersonlopez) [reply maquetado/login.html:19]: I used `hr` tag because if in the future we had to move the buttons the separator should be in the same place

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**